src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -118,6 +118,10 @@ string project
                 packageName: packageName, originalFileName: originalFilePath, codeFileName: reviewFilePath, originalFileStream: memoryStream,
                 project: project);
 
+            if (codeFile == null)
+            {
+                return StatusCode(statusCode: StatusCodes.Status204NoContent, $"API review code file for package {packageName} is not found in DevOps pipeline artifacts.");
```

Comment:
there is another exception on line 132, might be good to add a message to better know where it is coming from (or why)

Reply:
I will check and submit another.